tests/sentry/incidents/endpoints/test_organization_alert_rule_details.py

```diff
@@ -317,27 +317,24 @@ def test_dynamic_detection_type(self, mock_seer_request):
         resp = self.get_success_response(self.organization.slug, rule.id)
         assert rule.detection_type == resp.data.get("detectionType")
 
-        with pytest.raises(
-            ValidationError, match="Dynamic alerts require both sensitivity and seasonality"
-        ):
+        with pytest.raises(ValidationError, match="Dynamic alerts require a sensitivity level"):
             self.create_alert_rule(
                 seasonality=AlertRuleSeasonality.AUTO,
                 detection_type=AlertRuleDetectionType.DYNAMIC,
                 time_window=30,
             )  # Require both seasonality and sensitivity
 
-        with pytest.raises(
-            ValidationError, match="Dynamic alerts require both sensitivity and seasonality"
-        ):
-            self.create_alert_rule(
-                sensitivity=AlertRuleSensitivity.MEDIUM,
-                detection_type=AlertRuleDetectionType.DYNAMIC,
-                time_window=30,
-            )  # Require both seasonality and sensitivity
-
-        with pytest.raises(
-            ValidationError, match="Dynamic alerts require both sensitivity and seasonality"
-        ):
+        # NOTE: uncomment this test when seasonality becomes a supported field
```

Comment:
nit: s/NOTE/TODO

Reply:
You can keep it commented in but add a `pytest.skip` decorator